packages/core/src/types/framework.ts

```diff
@@ -18,8 +18,22 @@ export interface TestingFrameworkDriver {
    * Available testing framework API methods.
    */
   apiCatalog: TestingFrameworkAPICatalog;
+
+  /**
+   * Additional driver configuration options.
+   */
+  driverConfig: TestingFrameworkDriverConfig;
 }
 
+/**
+ * Additional driver configuration.
+ *
+ * @property shouldUseScreenSync - Indicates whether the driver should use wait for screen stability.
+ */
+export type TestingFrameworkDriverConfig = {
+  shouldUseScreenSync: boolean;
```

Comment:
`shouldUseScreenSync` → `useSnapshotStabilitySync`

Reply:
It explains more specifically what this boolean does (invokes screenshots comparison for sync)